opentelemetry-user-events-logs/src/lib.rs

```diff
@@ -133,6 +138,143 @@ mod tests {
         );
     }
 
+    #[ignore]
+    #[test]
+    fn integration_test_with_tracing() {
```

Comment:
Is earlier test subset of this - do we need them separate? This seems lots of repeated code in both tests, so checking.